htdocs/core/lib/bank.lib.php

```diff
@@ -113,7 +113,7 @@ function bank_prepare_head(Account $object)
     $head[$h][2] = 'info';
     $h++;*/
 
-	complete_head_from_modules($conf, $langs, $object, $head, $h, 'bank', 'remove');
+	//  complete_head_from_modules($conf, $langs, $object, $head, $h, 'bank', 'remove');
```

Comment:
The goal of this line is to allow an external module to modify the content of $head array (passed by reference) to removed an entry into tab. Asking modules if they want to remove a tab must be done only once all tabs are inserted into $head.